lib/main.js

```diff
@@ -56,7 +56,7 @@ function execShellCommand(cmd) {
 function run() {
     return __awaiter(this, void 0, void 0, function* () {
         try {
-            const ddevDir = core.getInput('ddevDir');
+            const ddevDir = core.getInput('ddevDir') || '.';
```

Comment:
I thought the default value in https://github.com/jonaseberle/github-action-setup-ddev/blob/master/action.yml#L14 was exactly to leave this out? 

Or did you do it to be on the safe side?

Reply:
That's only for our test runs where we call the script directly. It shouldn't be necessary in the normal use case according to https://github.com/actions/toolkit/issues/272#issuecomment-568516398